var/spack/repos/builtin/packages/libffi/package.py

```diff
@@ -12,10 +12,11 @@ class Libffi(AutotoolsPackage, SourcewarePackage):
     to call any function specified by a call interface description at
     run time."""
     homepage = "https://sourceware.org/libffi/"
-    sourceware_mirror_path = "libffi/libffi-3.2.1.tar.gz"
+    url      = "https://github.com/libffi/libffi/releases/download/v3.4.2/libffi-3.4.2.tar.gz"
```

Comment:
If we change the `url` does it make sense to keep the package still as a `SourcewarePackage`?

Reply:
Good point, totally missed this. Should be fixed now. GitHub seems to be the new way to get releases according to the website.